pycodestyle.py

```diff
@@ -1538,7 +1538,7 @@ def parse_udiff(diff, patterns=None, parent='.'):
             rv[path].update(range(row, row + nrows))
         elif line[:3] == '+++':
             path = line[4:].split('\t', 1)[0]
-            if path[:2] == 'b/':
+            if path[:2] in ('b/', 'w/', 'i/'):
```

Comment:
@mic4ael -- would you be able to add a comment here to explain where each of these values are coming from? That way we know why those are the magic values we want. As an aside, should this be "any character followed by a /" ? Something like: `if path[0].isalpha() and path[1] == '/':`

Additionally, if I could request that you add some tests for these cases into the test suite, then I think this should be mergeable (I don't see any issues, @sigmavirus24 , could you please be a second opinion?)

Reply:
I am not sure whether the approach of "any character followed by a /" is the right one since in `git` the list of used prefixes is limited (as far as i know). Sure, I will add the tests ;)